freeipa/src/main/java/com/sequenceiq/freeipa/sync/StackStatusCheckerJob.java

```diff
@@ -113,7 +115,11 @@ private void updateStackStatus(Stack stack, SyncResult result, List<ProviderSync
         DetailedStackStatus status = getStackStatus(providerSyncResults, result, alreadyDeletedCount);
         if (status != stack.getStackStatus().getDetailedStackStatus()) {
             if (autoSyncConfig.isUpdateStatus()) {
-                stackUpdater.updateStackStatus(stack, status, result.getMessage());
+                if (!flowLogService.isOtherFlowRunning(stack.getId())) {
```

Comment:
same here, code shouldn't reach this part if flow is running

Reply:
same answer as previous